unit_tests/test_ycsb_thread.py

```diff
@@ -133,16 +132,21 @@ def check_metrics():
         regex = re.compile(r'^collectd_ycsb_verify_gauge.*?([0-9\.]*?)$', re.MULTILINE)
 
         assert 'collectd_ycsb_verify_gauge' in output
-        assert 'UNEXPECTED_STATE' in output
+        assert 'type="UNEXPECTED_STATE"' in output
+        assert 'type="ERROR"' in output
         matches = regex.findall(output)
         assert all(float(i) >= 0 for i in matches), output
 
     check_metrics()
-    ycsb_thread2.get_results()
+    file_logger = get_events_logger(_registry=events.events_processes_registry)
```

Comment:
There is a shortcut for this function in EventsUtilsMixin. You can use it instead.

Reply:
fixed